internal/cli/streams/processor/describe.go

```diff
@@ -0,0 +1,93 @@
+package processor
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/usage"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20241113004/admin"
+)
+
+const jsonOutput = "json"
+
+type DescribeOpts struct {
+	cli.ProjectOpts
+	cli.OutputOpts
+	streamsInstance string
+	processorName   string
+	includeStats    bool
+	store           store.ProcessorDescriber
+}
+
+func (opts *DescribeOpts) Run() error {
+	describeParams := new(atlasv2.GetStreamProcessorApiParams)
+	describeParams.GroupId = opts.ProjectID
+	describeParams.TenantName = opts.streamsInstance
+	describeParams.ProcessorName = opts.processorName
+
+	r, err := opts.store.StreamProcessor(describeParams)
+	if err != nil {
+		return err
+	}
+
+	opts.Output = jsonOutput
+	if opts.includeStats {
+		return opts.Print(r)
+	}
+
+	sp := atlasv2.NewStreamsProcessorWithStats(r.Id, r.Name, r.Pipeline, r.State)
+	return opts.Print(sp)
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+// atlas streams processor describe <processorName>.
+func DescribeBuilder() *cobra.Command {
+	opts := &DescribeOpts{}
+	cmd := &cobra.Command{
+		Use:   "describe <processorName>",
```

Comment:
fwiw, this command seems like it would make more sense to have a json output file that is the .json configuration file that would work with the create command (and eventually update).